blog/04-4-2023-Build-a-dapp-for-buying-and-selling-unique-gadgets-on-celo-blockchain/Build-a-dapp-for-buying-and-selling-unique-gadgets-on-celo-blockchain.md

```diff
@@ -0,0 +1,905 @@
+---
+title: Build a dapp for buying and selling unique gadgets on celo blockchain
+description: Gadget project is a dApp for buying/selling unique gadgets represented by NFTs on Celo blockchain, with ERC-20 interface for DeFi. Provides secure/transparent platform for gadget enthusiasts/creators to connect.
+authors:
+  - name: Ikanji David
+    title: Technical Writer 
+    url:   https://github.com/Ikanji201
+    image_url:  https://avatars.githubusercontent.com/u/115812158?v=4
+tags: [celo sage, dapp, intermediate, celo]
+hide_table_of_contents: true
+slug: /tutorials/build-a-dapp-for-buying-and-selling-unique-gadgets-on-celo-blockchain
+---
+
+![build-a-dapp-for-buying-and-selling-unique-gadgets-on-celo-blockchain](https://user-images.githubusercontent.com/115812158/229753542-ef563aa6-d625-4327-8137-512020e43f97.png)
```

Comment:
Use this format for your header image instead.

![header](../../src/data-tutorials/showcase/intermediate/build-a-dapp-for-buying-and-selling-unique-gadgets-on-celo-blockchain.png)